exercises/house/house_test.go

```diff
@@ -1,19 +1,13 @@
-// Embed places a phrase with another phrase.
-//
-//    func Embed(prefixPhrase, suffixPhrase string) string
-
-// Verse generates a verse with relative phrases that have a recursive structure.
-//
-//    func Verse(prefixPhrase string, relPhrases []string, suffixPhrase string) string
-
-// As ever, there are different ways to do this. Try using Embed as a
-// subroutine of Verse, and using programmatic recursion to generate the return
-// value, reflecting the grammatical recursion of the song.
+// As ever, there are different ways to complete this exercise.
+// Try using using programmatic recursion to generate the verses of the song,
+// thus reflecting the song's grammatical recursion.
 
-// Song generates the full text of "The House That Jack Built". Of course, you
-// could return a string literal, but humor us; try using Verse as a subroutine.
+// While recursion isn't always the simplest or most efficient solution to a problem,
+// it's a powerful programming technique nonetheless.
+// New to recursion? Here are some useful articles to read up on it:
 //
-//    func Song() string
+// https://www.golang-book.com/books/intro/7#section5
+// https://www.goinggo.net/2013/09/recursion-and-tail-calls-in-go_26.html
```

Comment:
tbh with this link I mostly wanted to bring up the idea of tail call recursion. You're right it is a bit heavy for someone new to recursion, and as the article itself says, Go doesn't really optimise for it anyway. I'll remove it, it would turn up in a search for `golang + recursion` anyway if someone is interested in gaining a deeper understanding.